### DISCUSSION TITLE
Modules

### DISCUSSION THREAD

**Discussion by vlados** [livewire/livewire#5109] [3. Feature ideas]
I think for next version something like this will be very useful https://github.com/mhmiton/laravel-modules-livewire

**Comment by dcblogdev** [comment]
how do you mean? that repo adds support for generating Livewire components when using Laravel Modules package.

**Comment by joshhanley** [comment]
@vlados yeah can you expand on that? As Livewire does have support for putting components into module folders. Is something not working for you? Thanks!
